catboost/python-package/catboost/_catboost.pyx

```diff
@@ -112,6 +112,18 @@ class _NumpyAwareEncoder(JSONEncoder):
 class CatBoostError(Exception):
     pass
 
+
+class MultiLabelCustomMetric:
+    def evaluate(self):
```

Comment:
Let's enumerate all params here like in get_final_error method and write docstrings for all these parameters like [here](https://github.com/catboost/catboost/blob/master/catboost/python-package/catboost/core.py#L4507-L4520).
Don't forget to do the following:
```
@cython.embedsignature(True) 
class MultiLabelCustomMetric:
```